source/release_notes/3.4.13-2.11.rst

```diff
@@ -0,0 +1,50 @@
+.. _3.4.13-2.11:
+
+===================
+|PSMDB| 3.4.13-2.11
+===================
+
+:Date: March 5, 2018
+:Download: http://www.percona.com/downloads/percona-server-mongodb-3.4/
+:Installation: :ref:`install`
+
+|PSMDB| is an enhanced, open source, and highly-scalable database that is
+a fully-compatible, drop-in replacement for MongoDB 3.4 Community Edition.
+It supports MongoDB 3.4 protocols and drivers.
+
+|PSMDB| extends MongoDB Community Edition functionality by including the
+:ref:`inmemory` and :ref:`mongorocks` storage engines, as well as several
+enterprise-grade features.
+|PSMDB| requires no changes to MongoDB applications or code.
+
+This release is based on `MongoDB 3.4.13
+<https://docs.mongodb.com/manual/release-notes/3.4/#feb-10-2018>`_
+and includes the following additional change:
+
+* :jirabug:`PSMDB-191`: Fixed a bug in :ref:`mongorocks` engine initialization
+  code which caused wrong initialization of ``_maxPrefix`` value. This could
+  lead to reuse of dropped prefix and to accidental removal of data from the
+  collection using reused prefix.
+
+  In some specific conditions data records could disappear at arbitrary moment
+  of time from the collections or indexes created after server restart.
+
+  This could happen as the result of the following sequence of events:
+   1. User deletes one or more indexes or collections. These should be the ones
+         using maximum existing prefixes values.
+   2. User shuts down the server before :ref:`mongorocks` compaction thread
+         executes compactions of deleted ranges.
+   3. User restarts the server and creates new collections. Due to the bug
+         those new collections and their indexes may get the same prefix
+         values which were deleted and not yet compacted. User inserts some
+         data into the new collections.
+   4. After the server restart :ref:`mongorocks` compaction thread continues
+         executing compactions of the deleted ranges and this process may
+         eventually delete data from the collections sharing prefixes with
+         deleted ranges.
+
+* :jirabug:`PSMDB-164` MongoRocks would fails to repair if metadata was
+  inconsistent with dropped ``idents``. 
+
+* ``ServerStatus`` on MongoRocks is now accessing the storage engine without
```

Comment:
Should we mention upstream bug here? If not then probably we need to separate this sentence from previous bug (PSMDB-164). I was confused reading this because it looks like this sentence is the description of what was fixed for PSMDB-164.